LINUX_FAQ.md

```diff
@@ -0,0 +1,42 @@
+OpenDream RobustToolbox testing was done on Windows so I'm left to clean up.
+
+The first revision of this document is not intended as a formal guide, it's meant to help point out some things that need to be dealt with.
+
+It could be expanded to a formal guide though.
+
+Here's just some stuff you should know.
+
+*Some other information (written by Vera, PJB alerted me to this):* https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Client.CEF/CefManager.cs#L52
+
+## Compile errors involving RobustToolbox stuff
+
+You probably need to update RobustToolbox, cd over there, `git pull master`, `git submodule update --init --recursive`
+
+## Known-good CEF build
+
+`https://cef-builds.spotifycdn.com/cef_binary_91.1.6%2Bg8a752eb%2Bchromium-91.0.4472.77_linux64_beta_minimal.tar.bz2`
```

Comment:
this is hillarious

Reply:
...hey, it worked for me

Reply:
It's either Spotify's CEF builds or spend-two-days-compiling-from-source. Pick one.